3/debian-10/rootfs/opt/bitnami/scripts/libzookeeper.sh

```diff
@@ -247,14 +257,26 @@ zookeeper_generate_conf() {
     zookeeper_conf_set "${ZOO_CONF_DIR}/log4j.properties" zookeeper.console.threshold "$ZOO_LOG_LEVEL"
 
     # Add zookeeper servers to configuration
+    idFlag=false
+    if [[ "$ZOO_SERVERS" == *"::"* ]]; then
+        idFlag=true
+    fi
     read -r -a zookeeper_servers_list <<< "${ZOO_SERVERS//[;, ]/ }"
     if [[ ${#zookeeper_servers_list[@]} -gt 1 ]]; then
-        local i=1
-        for server in "${zookeeper_servers_list[@]}"; do
-            info "Adding server: ${server}"
-            zookeeper_conf_set "$ZOO_CONF_FILE" server.$i "${server};${ZOO_PORT_NUMBER}"
-            (( i++ ))
-        done
+        if [[ "${idFlag}" == "false" ]]; then
+            local i=1
+            for server in "${zookeeper_servers_list[@]}"; do
+                info "Adding server: ${server}"
+                zookeeper_conf_set "$ZOO_CONF_FILE" server.$i "${server};${ZOO_PORT_NUMBER}"
+                (( i++ ))
+            done
+        else
+            for server in "${zookeeper_servers_list[@]}"; do
+                read -r -a SRV <<< "${server//::/ }"
+                info "Adding server: ${SRV[0]} with id: ${SRV[1]} "
+                zookeeper_conf_set "$ZOO_CONF_FILE" server.${SRV[1]} "${SRV[0]};${ZOO_PORT_NUMBER}"
```

Comment:
Please do not use uppercase for the "SRV" variable:
```suggestion
                read -r -a srv <<< "${server//::/ }"
                info "Adding server: ${srv[0]} with id: ${srv[1]} "
                zookeeper_conf_set "$ZOO_CONF_FILE" server.${srv[1]} "${srv[0]};${ZOO_PORT_NUMBER}"
```